tests/cases/fourslash/findAllRefsShorthandPropertyFlag.ts

```diff
@@ -0,0 +1,56 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: /file1.ts
+////declare function log(s: string | number): void;
+////const [|{| "isWriteAccess": true, "isDefinition": true |}q|] = 1;
+////export { [|{| "isWriteAccess": true, "isDefinition": true |}q|] };
+////const x = {
+////    [|{| "isWriteAccess": true, "isDefinition": true |}z|]: 'value'
+////}
+////const { [|{| "isWriteAccess": true, "isDefinition": true |}z|] } = x;
+////log([|z|]);
+
+// @Filename: /file2.ts
+////declare function log(s: string | number): void;
+////import { [|{| "isWriteAccess": true, "isDefinition": true |}q|] } from "./file1";
+////log([|q|] + 1);
+
+verify.noErrors();

```

Comment:
Is the name of this file referring to the prefix/suffix flag?

Reply:
Renamed test file to be clearer